example/plugins/frontends/oidc_op_frontend.yaml.example

```diff
@@ -0,0 +1,257 @@
+          kwargs:
+            grant_config:
+              expires_in: 43200
+              usage_rules:
+                access_token: {}
+                authorization_code:
+                  max_usage: 1
+                  supports_minting:
+                  - access_token
+                  - refresh_token
+                  - id_token
+                refresh_token:
+                  supports_minting:
+                  - access_token
+                  - refresh_token
+        capabilities:
+          grant_types_supported:
+          - authorization_code
+          - urn:ietf:params:oauth:grant-type:jwt-bearer
+          - refresh_token
+          subject_types_supported:
+          - public
+          - pairwise
+        endpoint:
+          provider_info:
+            class: oidcop.oidc.provider_config.ProviderConfiguration
+            kwargs:
+              client_authn_method: null
+            path: .well-known/openid-configuration
+          authorization:
+            class: oidcop.oidc.authorization.Authorization
+            kwargs:
+              claims_parameter_supported: true
+              client_authn_method: null
+              request_object_encryption_alg_values_supported: &id001
+              - RSA-OAEP
+              - RSA-OAEP-256
+              - A192KW
+              - A256KW
+              - ECDH-ES
+              - ECDH-ES+A128KW
+              - ECDH-ES+A192KW
+              - ECDH-ES+A256KW
+              request_parameter_supported: true
+              request_uri_parameter_supported: true
+              response_modes_supported:
+              - query
+              - fragment
+              - form_post
+              response_types_supported:
+              - code
+            path: OIDC/authorization
+          token:
+            class: oidcop.oidc.token.Token
+            kwargs:
+              client_authn_method:
+              - client_secret_post
+              - client_secret_basic
+              - client_secret_jwt
+              - private_key_jwt
+            path: OIDC/token
+          userinfo:
+            class: oidcop.oidc.userinfo.UserInfo
+            kwargs:
+              claim_types_supported:
+              - normal
+              - aggregated
+              - distributed
+              userinfo_encryption_alg_values_supported: *id001
+              userinfo_signing_alg_values_supported: &id002
+              - RS256
+              - RS512
+              - ES256
+              - ES512
+              - PS256
+              - PS512
+            path: OIDC/userinfo
+          introspection:
+            class: oidcop.oauth2.introspection.Introspection
+            kwargs:
+              client_authn_method:
+              - client_secret_post
+              - client_secret_basic
+              - client_secret_jwt
+              - private_key_jwt
+              release:
+              - username
+            path: OIDC/introspection
+          #registration:
```

Comment:
dynamic client registration, registration api and endsession endpoints (logout) was not implemented at this time.

Reply:
Implemented!